ui/perfherder/constants.js

```diff
@@ -19,6 +19,12 @@ export const filterText = {
   inputPlaceholder: 'filter text e.g. linux tp5o',
 };
 
+export const selectorCardText = {
```

Comment:
Since you're moving the messages here to presumably make them reusable, it'd be worth giving the `const` a more generic name.